src/libcommon/keychainmanager/apitoken.cpp

```diff
@@ -62,4 +62,18 @@ ApiToken::ApiToken(const std::string &data) : _rawData(data) {
     }
 }
 
+std::string ApiToken::reconstructJsonString() {
+    Poco::JSON::Object obj;
```

Comment:
It would be nice to have a unit for this method.

Reply:
Done. libCommon had no test until now. I have added the module in CMakeList

Reply:
Thanks!